A2017/tensorflow.ipynb

```diff
@@ -0,0 +1,1357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow tutorial\n",
+    "_MILA, November 2017_\n",
+    "\n",
+    "## Protip: browsing through the TensorFlow API\n",
+    "\n",
+    "The [devdocs.io](http://devdocs.io/) website is an amazing resource to browse through the TensorFlow Python API (as well as many other APIs such as numpy or the Python API itself).\n",
+    "\n",
+    "# Using TensorFlow at MILA\n",
+    "\n",
+    "The most straightforward way to access TensorFlow using the MILA software stack is through the `TODO: WRITEME` conda environment. To activate the `TODO: WRITEME` environment, use the command\n",
+    "\n",
+    "```bash\n",
+    "source activate TODO: WRITEME\n",
```

Comment:
Hi @vdumoulin you still have `todo: writeme` here and above (line 16). Is it expected ?